### PR TITLE
Alex test uploaded

### PR DESCRIPTION
**Pull request by Ozleblebici**
Local test completed successfully, ready for review 
Ticket : TC0212 - Bla Bla mode selection

image

### CODE REVIEW

File: src/test/java/Test1.java
Comment (Tirkish89) [inline src/test/java/Test1.java:1]: looks good

**Review by ATABEG00 — APPROVED** [review]
Good job!

**Review by Tirkish89 — APPROVED** [review]

File: src/test/java/Test1.java
Comment (Ozleblebici) [reply src/test/java/Test1.java:1]: danke

**Comment by 1erkan1** [comment]
very good

**Comment by Gylych1110** [comment]
very well

**Review by 1erkan1 — APPROVED** [review]

**Pull request merged**